libs/pika/async_mpi/tests/unit/CMakeLists.txt

```diff
@@ -4,8 +4,9 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-set(tests # algorithm_transform_mpi
-    mpi_ring_async_sender_receiver
+set(tests #
+    algorithm_transform_mpi #
+    mpi_ring_async_sender_receiver #
```

Comment:
Do we need the `#`s?

Reply:
they stop cmake-format from sticking everything on one line. I don't like our cmake-format rules very much ;)

Reply:
I kind of guessed that was the reason. You know we can change the configuration, right? In particular, I think this one controls how this is wrapped: https://github.com/pika-org/pika/blob/0803db56fb5a32920b6563149a32f674a5bdc5d6/.cmake-format.py#L46. Setting it to `1` would put these elements all on separate lines.

Why don't revert this change and open a separate PR with the `cmake-format.py` changes you'd propose, then we can discuss there?